metagraph/src/common/seq_tools/reverse_complement.hpp

```diff
@@ -28,38 +28,38 @@
 
 #include <htslib/kseq.h>
 
+static const unsigned char kCompTab[] = {
```

Comment:
`kCompTab ` looks like it's a scalar constant.

Let's call it COMPL_TAB?

Reply:
done